WcaOnRails/app/webpacker/components/CompetitionForm/Inputs/FormSearch.js

```diff
@@ -84,6 +85,7 @@ export function CompetitionSearch({
     <WcaSearch
       value={compOptions}
       onChange={onChangeIdOnly}
+      models={[SEARCH_MODELS.competition]} // TODO: Need to check if this is needed.
```

Comment:
What do you mean by "if this is needed"?

Reply:
Actually wanted to check with you whether this is needed. Because I was trying to replace `model` with `models` and found this `WcaSearch which doesn't even have `model`. I don't think this feature currently works because there is no model. But I didn't test this as well, because I'm not aware of the exact process to create series competition (or in other words I didn't try to do it).

I assume this is needed, and removed the TODO comment. If you think this is fine, you can resolve it, else let me know what change needs to be made.